subsys/bluetooth/controller/ll_sw/ull_scan_aux.c

```diff
@@ -511,6 +512,22 @@ void ull_scan_aux_setup(memq_link_t *link, struct node_rx_pdu *rx)
 		goto ull_scan_aux_rx_flush;
 	}
 
+	/* Determine the window size */
+	if (aux_ptr->offs_units) {
+		window_size_us = OFFS_UNIT_300_US;
+	} else {
+		window_size_us = OFFS_UNIT_30_US;
+	}
+
+	/* Calculate received aux offset we need to have ULL schedule a reception */
+	aux_offset_us = (uint32_t)PDU_ADV_AUX_PTR_OFFSET_GET(aux_ptr) * window_size_us;
```

Comment:
Here you multiple 2 `uint32_t` and store the result in another `uint32_t`. Should have some sort of overflow protection

Reply:
The typecast sounds OK but why is `window_size_us` defined as a `uint32_t` then? 

Since `window_size_us` is either `OFFS_UNIT_300_US` or `OFFS_UNIT_30_US`, then it looks like it can be a `uint16_t` and thus the line can become
```suggestion
	/* Calculate received aux offset we need to have ULL schedule a reception */
	aux_offset_us = ((uint16_t)PDU_ADV_AUX_PTR_OFFSET_GET(aux_ptr)) * window_size_us;
```
or maybe even just
```suggestion
	/* Calculate received aux offset we need to have ULL schedule a reception */
	aux_offset_us = PDU_ADV_AUX_PTR_OFFSET_GET(aux_ptr) * window_size_us;
```

It's a minor detail, and if you don't think `window_size_us` should be a `uint16_t`, then let me know and I can approve

Reply:
Used `uint16_t` for `window_size_us`, and keeping the `(uint32_t)` integer promotion typecast for 16-bit big-endian CPUs.